scripts/imgtool/keys/ecdsa.py

```diff
@@ -102,37 +102,57 @@ def generate():
     def _get_public(self):
         return self.key.public_key()
 
-    def _build_minimal_ecdsa_privkey(self, der):
+    def _build_minimal_ecdsa_privkey(self, der, format):
+        if (format == serialization.PrivateFormat.OpenSSH):
+            print(os.path.basename(__file__) + \
+                ": Warning: --minimal is supported only for PKCS8 or TraditionalOpenSSL formats")
+            return bytearray(der)
         '''
         Builds a new DER that only includes the EC private key, removing the
         public key that is added as an "optional" BITSTRING.
         '''
```

Comment:
This was a docstring for the function, if you want to leave it here changed it to comments.

Reply:
It was done by error, moved it back to the correct place.